core/test/audits/dobetterweb/doctype-test.js

```diff
@@ -72,10 +90,11 @@ describe('DOBETTERWEB: doctype audit', () => {
     expect(auditResult.explanation).toBeDisplayString('Expected systemId to be an empty string');
   });
 
-  it('succeeds when document contains a doctype, and the name value is "html"', () => {
+  // eslint-disable-next-line max-len
+  it('succeeds when document is CSS1Compat, did not detect limit-quirks-mode, and regardless of doctype values seen', () => {
```

Comment:
this test is not very realistic, any better ideas?

Reply:
Maybe use the case from the reported issue